sdk/dotnet/Provider.cs

```diff
@@ -1,4 +1,4 @@
-// *** WARNING: this file was generated by the Pulumi SDK Generator. ***
+// *** WARNING: this file was generated by pulumigen. ***
```

Comment:
I don't love this.  We may want to undo the part of the change in https://github.com/pulumi/pulumi-provider-boilerplate/pull/11/files#diff-edba8f02af121e983e5788bc4949c74194a91b7a212cfec8617c4a1799c8ae77L53 that changed this.  We may also want to align with back up into the native providers where I assume this came from?